src/brainbay-DESKTOP-1VOANM7.rc

```diff
@@ -0,0 +1,3022 @@
+// Microsoft Visual C++ generated resource script.
+//
+#include "resource.h"
```

Comment:
is this file necessary ? seems to be a local build product .. ?

Reply:
Good catch Chris. I've deleted this file.